blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md

```diff
@@ -0,0 +1,501 @@
+---
+title: Connect and interact with celo using web3Modal
+description: Alternative method of connecting to Celo networks
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [advanced, celosage, solidity, react]
+hide_table_of_contents: true
+slug: /tutorials/connect-and-interact-with-celo-using-webmodal
+---
+
+../../src/data-tutorials/showcase/advanced/connect-and-interact-with-celo-using-webmodal.png
```

Comment:
`here you still need the ![header](../../src/data-tutorials/showcase/advanced/connect-and-interact-with-celo-using-webmodal.png)`